types/api/api.d.ts

```diff
@@ -0,0 +1,9 @@
+export declare class API {
```

Comment:
Should probably use `unknown` as the return type. Do you see any reason why not?

Reply:
Its better to be `unknown`, ive made the change.

Reply:
Please push that so I can approve. You can add the stricter types later.